### PR TITLE
Exp aws example platform

### PR DESCRIPTION
**Pull request by haim-kermany**
creating new file 
moved some code to common vpc

### CODE REVIEW

**Review by ShiriMoran — APPROVED** [review]

File: pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+Details:
+~~~~~~~~
+Path is enabled; The relevant rules are:
+	Egress:
+		security group GroupName:11 allows connection with the following allow rules
+			index: 1, direction: outbound,  conns: protocol: all, target: 0.0.0.0/0
+		network ACL NetworkAclId:48 allows connection with the following allow rules
+			index: 0, direction: outbound ,cidr: 0.0.0.0/0, conn: -1, action: allow
```
Comment (ShiriMoran) [inline pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt:20]: What is conn -1?

File: pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Explaining connectivity from 10.240.40.184 to 10.240.20.141 within vpc0
+Interpreted source: InstanceId:7[10.240.40.184]
+Interpreted destination: InstanceId:15[10.240.20.141]
+=======================================================================
+
+Connections from InstanceId:7[10.240.40.184] to InstanceId:15[10.240.20.141]: All Connections
+
+Path:
+	InstanceId:7[10.240.40.184] -> security group GroupName:11 -> network ACL NetworkAclId:48 -> dashoard ->
```
Comment (ShiriMoran) [inline pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt:9]: dashoard -> dashboard ?

File: pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt [hunk 3]
```diff
@@ -0,0 +1,38 @@
+Explaining connectivity from 10.240.40.184 to 10.240.20.141 within vpc0
+Interpreted source: InstanceId:7[10.240.40.184]
+Interpreted destination: InstanceId:15[10.240.20.141]
+=======================================================================
+
+Connections from InstanceId:7[10.240.40.184] to InstanceId:15[10.240.20.141]: All Connections
+
+Path:
+	InstanceId:7[10.240.40.184] -> security group GroupName:11 -> network ACL NetworkAclId:48 -> dashoard -> 
+	application -> network ACL NetworkAclId:48 -> security group GroupName:18 -> InstanceId:15[10.240.20.141]
```
Comment (ShiriMoran) [inline pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt:10]: 10.240.40.184 and 10.240.20.141 do not share the same subnet.
The former has SubnetId:6 and the latter SubnetId:14
Why aren't the subnets reflected at the path?

File: pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt (hunk 1 above)
Comment (haim-kermany) [reply pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt:20]: I added one explain example, w/o debugging it, just for start fixing...

File: pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt (hunk 2 above)
Comment (haim-kermany) [reply pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt:9]: I added one explain example, w/o debugging it, just for start fixing...

File: pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt (hunk 3 above)
Comment (haim-kermany) [reply pkg/awsvpc/examples/out/explain_out/ip_to_ip_all_vpcs_explain_detail.txt:10]: I added one explain example, w/o debugging it, just for start fixing...

**Pull request merged**